changelog/6601.bugfix.md

```diff
@@ -0,0 +1,2 @@
+Fixed a bug in the featurization of the boolean slot type. Previously, to set a slot value to "true", 
+you had to set it to "1", which is in conflict with the documentation.
```

Comment:
You didn't have to set it to `"1"`, just to `true` (no quotes) AFAIK
```
- slot{"my_bool_slot": true}
```

Reply:
Ok, but that's still confusing users.

Reply:
I guess I make this an `improvement` then, instead of a bug fix? Also, I am not sure about the `float(x) == 1.0`. Perhaps it'd be better to have `float(x) != 0.0` as before, though that's a bit odd I think.

Reply:
Oh I totally agree that this is a good improvement! (or bugfix if you expected `"true"` to work). Just wanted to point out that saying you had to set it to 1 would probably confuse a lot of people whose bots were working as expected before.

Reply:
@erohmensing just FYI that while just `true` might work for training time, I think there might be an issue with the JSON validation schema since I got [this](https://forum.rasa.com/t/questions-regarding-data-validation-script-and-with-slots-entites-and-followups-from-custom-actions/33405/8?u=nivek) error message when changing stories to use just `true`.

Reply:
I can confirm this error.